gapii/cc/spy.cpp

```diff
@@ -460,6 +486,22 @@ void Spy::gvr_frame_submit(CallObserver* observer, gvr_frame** frame,
   GvrSpy::gvr_frame_submit(observer, frame, list, head_space_from_start_space);
 }
 
+bool Spy::checkEndTrace() {
+  if (!is_suspended() && mCaptureFrames < 0) {
```

Comment:
Sould this be `mCaptureFrames <= 0` ?

Reply:
No, mCaptureFrames == 0 means continue until manual stop, -1 means end of trace.
I have added a comment to the variable declaration.

Reply:
Thanks for clarifying this.